### DISCUSSION TITLE
컨벤션 관련 제안입니다!

### DISCUSSION THREAD

**Discussion by BearCrying** [sixes-sense/back-end#2] [Ideas]
서로서로 작성한 코드를 이해하려면, 전체적으로 코드 양식을 통일화 할 필요가 있을 것 같다는 생각이들었습니다 ㅎㅎ.. 
그래서 적당한 규칙을 기존에 쓰던것을 베이스로 수정해서 작성했습니다

한번 읽어보시고 괜찮은 것 같으면 적용하면 좋을 것 같습니다 :)

질문 있으면 댓글로 달아두시고, 수업이 끝나고 회의할 때 잠깐 이야기해보면 좋을 것 같아요!

아래 저희 노션 링크 달아두겠습니다
https://www.notion.so/hammang/35b37952da2f4b2a883a0b303ab87b76

**Comment by myrhymetree** [comment]
넵 확인했습니다 커밋규칙관련의 git squash부분은 좀 번거롭지 않을까 싶네요
